Documentation/Contributing/development-flow.md

```diff
@@ -287,9 +287,10 @@ Signed-off-by: First Name Last Name <email address>
 
 The `component` **MUST** be in the [list checked by the CI](https://github.com/rook/rook/blob/master/.commitlintrc.json).
 
-Note: sometimes you will feel like there is not so much to say, for instance if you are fixing a typo in a text.
-In that case, it is acceptable to shorten the commit message.
-Also, you don't always need to close an issue, again for a very small fix.
+!!! note
+    sometimes you will feel like there is not so much to say, for instance if you are fixing a typo in a text.
```

Comment:
We need to start the sentence for the note with a capital letter.
```suggestion
    Sometimes you will feel like there is not so much to say, for instance if you are fixing a typo in a text.
```

Reply:
@travisn I am facing issues with codespell check. Could you please guide me through the issue.

Reply:
@YashasviChaurasia I'll create the pr to fix that codespell ci, it is strange that it fails on random pr changes, it is also showing on my pr. For now, I'll suggest if the codespell error in not coming from you changes overlook that